### ISSUE TITLE
Hi, fellow Goon

### ISSUE THREAD

**Issue by zoinksbob** [RobertZFord/FreeRepublic#1]
Hi. I got here from the Freep thread on SomethingAwful. I've been training an LSTM on posts from Freep, and I wrote my own C# program to help harvest posts and turn them into an appropriate format for training. My program takes a pretty different approach from yours, though. I've been looking over your code and thinking how to use the best of both our programs.

Is it cool if start making a few commits to your code? To start with, for example, I found and fixed a small bug where your program fails on threads with more than 1000 replies.

**Comment by RobertZFord** [comment]
Yeah dude, go wild!  I wrote this as a toy library while watching some TV.  Do whatever you want with it, make whatever commits/pulls/etc.  Hopefully I won't screw up the repo too badly.  I've never used Github before but that's what all the cool kids were using so I figured I would jump on the bandwagon.

**Comment by zoinksbob** [comment]
Cool. I think you have to add me as a contributor before I can make commits. I haven't used git much either, but I'm going to need to learn it for work anyway, so this could be good practice.

**Comment by RobertZFord** [comment]
There you go.  I added you as a collaborator.  Let me know if that works.
